plugins/woocommerce-blocks/assets/js/base/components/cart-checkout/form/form.tsx

```diff
@@ -149,6 +140,34 @@ const Form = < T extends AddressFormValues | ContactFormValues >( {
 					);
 				}
 
+				// Handle both address_1 and address_2 fields together.
+				if ( field.key === 'address_1' ) {
+					return (
+						<AddressFields
+							id={ id }
+							addressType={ addressType }
+							key={ field.key }
+							address1={ addressFormFields.fields.find(
+								( customField ) =>
+									customField.key === 'address_1'
+							) }
+							address2={ addressFormFields.fields.find(
+								( customField ) =>
+									customField.key === 'address_2'
+							) }
+							addressFormFields={ addressFormFields }
+							fieldsRef={ fieldsRef }
+							onChange={ onChange }
```

Comment:
There is a TS error here:

`TS2322: Type ( newValue: T ) => void is not assignable to type
( newValue: AddressFormValues | ContactFormValues ) => void`

Reply:
Good catch, and just solved.